src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,21 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany
     private List<Smile> smiles;
```

Comment:
I think should use @ManyToMany, because smile can be on many comments

Reply:
it was said 
"One comment can have multiple smiles" in README.
so i did it so
let we ask for mentor review for this point?
may be, both versions are possible depending on business logic?

Reply:
ok